src/test/resources/Features/APIDocumentationUpdate.feature

```diff
@@ -0,0 +1,12 @@
+Feature: API Documentation updates
+         OCD-2505 - Update Implementation Note for /certified_products endpoint
+
+Scenario Outline: Verify that implementation note for /certified_products endpoint is updated with new text
+    Given I am on CHPL API page
+    When I click on "<controller_name>" to see operations list
+    And I click on Certified Products endpoint link to view details
```

Comment:
Either the "When" and "Then" steps shouldn't be parameterized, or the "And" step should. There's no way to do a second Example without changing the "And".

The "solves the feature" option would be to not parameterize either of them, while the "prepares for the future" option would be to do it for them all. If you _do_ do it for them all, I'd recommend putting an Example in that should have been successful even before the story started